src/DroneSafetyApi/Data/DbOptions.cs

```diff
@@ -5,10 +5,22 @@
 
 namespace DroneSafetyApi.Data
 {
+    /// <summary>
+    /// The DbOptions class is a data model to represents configurations for a DocumentDb query
+    /// </summary>
     public class DbOptions
     {
+        /// <summary>
+        /// The EndpointUri property represents the the endpoint URI of the DocumentDB database
```

Comment:
Do you have to specify "The EndpointUri property..." or can you just say "represents..."